src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -459,7 +530,214 @@ tests:
+
+          Note: The number of ExtensionFieldSets, the value of clusterId of each ExtensionFieldSet, the number of attributes in attributeValueList and their values varies for each application
+
+          Verify the "status is success" on the TH(Chip-tool)  Log  and below is the sample log provided for the raspi platform:
+          [1670970505.887060][5742:5744] CHIP:DMG: InvokeResponseMessage =
+          [1670970505.887118][5742:5744] CHIP:DMG: {
+          [1670970505.887172][5742:5744] CHIP:DMG: 	suppressResponse = false,
+          [1670970505.887231][5742:5744] CHIP:DMG: 	InvokeResponseIBs =
+          [1670970505.887307][5742:5744] CHIP:DMG: 	[
+          [1670970505.887366][5742:5744] CHIP:DMG: 		InvokeResponseIB =
+          [1670970505.887444][5742:5744] CHIP:DMG: 		{
+          [1670970505.887507][5742:5744] CHIP:DMG: 			CommandDataIB =
+          [1670970505.887576][5742:5744] CHIP:DMG: 			{
+          [1670970505.887713][5742:5744] CHIP:DMG: 				CommandPathIB =
+          [1670970505.887804][5742:5744] CHIP:DMG: 				{
+          [1670970505.887940][5742:5744] CHIP:DMG: 					EndpointId = 0x1,
+          [1670970505.888093][5742:5744] CHIP:DMG: 					ClusterId = 0x5,
+          [1670970505.888242][5742:5744] CHIP:DMG: 					CommandId = 0x0,
+          [1670970505.888385][5742:5744] CHIP:DMG: 				},
+          [1670970505.888692][5742:5744] CHIP:DMG:
+          [1670970505.888769][5742:5744] CHIP:DMG: 				CommandFields =
+          [1670970505.888852][5742:5744] CHIP:DMG: 				{
+          [1670970505.889030][5742:5744] CHIP:DMG: 					0x0 = 0,
+          [1670970505.889183][5742:5744] CHIP:DMG: 					0x1 = 1,
+          [1670970505.889406][5742:5744] CHIP:DMG: 					0x2 = 1,
+          [1670970505.889515][5742:5744] CHIP:DMG: 				},
+          [1670970505.889603][5742:5744] CHIP:DMG: 			},
+          [1670970505.889684][5742:5744] CHIP:DMG:
+          [1670970505.889745][5742:5744] CHIP:DMG: 		},
+          [1670970505.889821][5742:5744] CHIP:DMG:
+          [1670970505.889878][5742:5744] CHIP:DMG: 	],
+          [1670970505.889953][5742:5744] CHIP:DMG:
+          [1670970505.890009][5742:5744] CHIP:DMG: 	InteractionModelRevision = 1
+          [1670970505.890066][5742:5744] CHIP:DMG: },
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: S.S.C00.Rsp && PICS_SKIP_SAMPLE_APP
+      arguments:
+          values:
+              - name: "message"
+                value:
+                    "Please execute the add scene command with
+                    extensionfieldsets in accordance with AC1 on DUT and enter
+                    'y' if the command is successful"
+              - name: "expectedValue"
+                value: "y"
+
+    - label:
+          "Step 6b: TH sends a ViewScene command to DUT with the GroupID field
+          set to G1 and the SceneID field set to 0x03."
+      PICS: S.S.C01.Rsp && PICS_SDK_CI_ONLY
```

Comment:
Same question - what's missing to make this automatable for real?

Reply:
We would need to select the AC depending on mandatory clusters on the DUT, the scenable attributes on those clusters and the proper verification commands to confirm these attributes are properly restored when switching scenes.